doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,81 @@
+# Advocacy Ambassador program
+
+The project aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up to date set of messages that it would like abassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
+* Maintaining an up to date set of topics on which additional content
+  would benefit the community. These will be documented in the
+  nodejs/ambassadors repository.
+* Providing a space for collaborators to share new topics/features they
+  are working on, when they would be happy for ambassadors to help share and get
+  early feedback.
+* Helping ambassadors promote content that aligns with the
+  messages and topics defined through social media and other channels.
+* Advocating for ambasadors to be part of the OpenJS speakers bureau, even if the
```

Comment:
unable to follow this statement (blame it on my lack of understanding). would you pls explain? does it mean openjs speakers bureau should take lead in advocacy, or they should identify ambassadors? or something else?

Reply:
thanks @mhdawson , understood now, no suggestions; the text is good as is.